plugins/Jobs.cpp

```diff
@@ -25,6 +25,25 @@ class scopedDisableNoopMode {
     bool _wasNoop;
 };
 
+int64_t BedrockPlugin_Jobs::getNextID(SQLite& db)
```

Comment:
nab, seems like this needs to be moved into the base class since we are using it multiple places now

Reply:
I don't think so. Yeah, we did the same thing in two places, but they're not related to each other at all.